fsgrid.hpp

```diff
@@ -1040,3 +1086,108 @@ template <typename T, int stencil> class FsGrid : public FsGridTools{
          array[2] = a;
       }
 };
+
+
+//Array Operator Overloads
```

Comment:
Heh. Aren't these more or less exactly what Markus was talking about for the GPU-Implementation's data structure needs?

Reply:
Yep that was part of my thinking